src/main/java/webApp/ServingWebContentApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages={"repository"})
+@EntityScan(basePackages = {"repository"})
```

Comment:
maybe have the word repository as a constant as it is used very often as a plain string